internal/quickstart/show_sdk_instructions.go

```diff
@@ -98,6 +105,10 @@ func (m showSDKInstructionsModel) Init() tea.Cmd {
 		cmds = append(cmds, fetchEnv(m.environmentsClient, m.accessToken, m.baseUri, defaultEnvKey, defaultProjKey))
 	}
 
+	if m.sdkKind == clientSideSDK {
```

Comment:
interestingly all the flags we create using the go sdk [have mobile keys enabled by default](https://github.com/launchdarkly/api-client-go/blob/ea6705ab8a49db1755fdcc47ad60e04cec85eed9/model_client_side_availability_post.go#L19-L24)